packages/hoppscotch-desktop/src-tauri/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hoppscotch-desktop"
-version = "25.8.0"
+version = "25.9.0"
```

Comment:
Version bumps for the release were already handled in https://github.com/hoppscotch/hoppscotch/commit/6ae85ea9fa5456b46382c5d6511ea6a0aa4a4caf.